README.md

```diff
@@ -17,7 +17,11 @@ Role Variables
 On Mac OSX and Windows, use e.g. `gitlab_runner_wanted_version: 12.4.1`.
 On Linux, use `gitlab_runner_package_version` instead.
 - `gitlab_runner_concurrent` - The maximum number of global jobs to run concurrently. Defaults to the number of processor cores.
-- `gitlab_runner_registration_token` - The GitLab registration token. If this is specified, a runner will be registered to a GitLab server.
+- `gitlab_runner_registration_token` - The GitLab registration token. If this is specified, each runner will be registered to a GitLab server. Deprecating in gitlab version 16.0 and removed in 18.0.
+- `gitlab_runner_registration_token_type` - Gitlab runner registration token type
+Set "runner-token" to register runner with --token option (new workflow https://docs.gitlab.com/ee/ci/runners/new_creation_workflow.html)
```

Comment:
The wording in Gitlab is always 
* Authentication Token (for the new workflow)
* Registration Token (for the old workflow)

I don't know if "runner-tokne" is also a common naming. I'd stick to the two mostely used in the gitlab documentation